protected/humhub/modules/content/components/ContentContainerDefaultPermissionManager.php

```diff
@@ -28,9 +28,21 @@ class ContentContainerDefaultPermissionManager extends PermissionManager
      */
     protected function getModulePermissions(\yii\base\Module $module)
     {
+        if ($module instanceof ContentContainerModule && method_exists($module, 'getContainerPermissions')) {
```

Comment:
Think we can remove the `method_exists` here since isntanceof should make sure this method exists, hiding it in sub classes should not be allowed.

Reply:
https://github.com/humhub/humhub/commit/2059b55bc78fe127acfb0806b1e1c9df3a5e996e